book/src/user/experimental-features.md

```diff
@@ -4,11 +4,7 @@ In `privacy-scaling-explorations/halo2` fork we have implemented many experiment
 
 ## Commitment scheme abstraction
 
-To support different kinds of polynomial commitment schemes, we've added a trait `CommitmentScheme` to allow create/verify proofs with different commitment scheme implementations, currently there are 2 available implementations in this fork:
-
-- [`IPACommitmentScheme`](https://privacy-scaling-explorations.github.io/halo2/halo2_proofs/poly/ipa/commitment/struct.IPACommitmentScheme.html)
-
-  The original implementation from `zcash/halo2` with the original multi-open strategy `{Prover,Verifier}IPA`
```

Comment:
Hmm, I think this means that we keep the `CommitmentScheme` abstraction. 
I think we plan to remove this abstraction. no?